src/media/media.service.ts

```diff
@@ -0,0 +1,58 @@
+import { getSignedCookies } from '@aws-sdk/cloudfront-signer';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { InjectEntityManager } from '@nestjs/typeorm';
+import dayjs from 'dayjs';
+import { EntityManager } from 'typeorm';
+import { MediaInfrastructure } from './media.infra';
+
+@Injectable()
+export class MediaService {
+  private readonly awsCloudfrontKeyPairId: string;
+  private readonly awsCloudfrontPrivateKey: string;
+
+  constructor(
+    @InjectEntityManager() private readonly entityManager: EntityManager,
+    private readonly mediaInfra: MediaInfrastructure,
+    private readonly configService: ConfigService<{
+      AWS_CLOUDFRONT_KEY_PAIR_ID: string;
+      AWS_CLOUDFRONT_PRIVATE_KEY: string;
```

Comment:
These keys is not add into `.env` & `.env.example`  .

Reply:
fixed